dart/biomechanics/MarkerFitter.cpp

```diff
@@ -5758,7 +5859,8 @@ std::shared_ptr<SphereFitJointCenterProblem> MarkerFitter::findJointCenter(
   for (int i = 0; i < mJointSphereFitSGDIterations; i++)
   {
     Eigen::VectorXs grad = problem->getGradient();
-    Eigen::VectorXs newAccum = accum + grad.cwiseProduct(grad);
+    Eigen::VectorXs newAccum
+        = decay * accum + (1.0 - decay) * grad.cwiseProduct(grad);
```

Comment:
Is `decay` supposed to be iteratively updated? If not, this sum is now much more biased towards the current value of `accum`. Maybe that's intentional, just trying to understand.